frontend/components/organisms/CourseList.tsx

```diff
@@ -19,17 +22,80 @@ import { BsChevronRight } from 'react-icons/bs'
 import { CourseType } from '../../types'
 import { Loader } from '../atoms/Loader'
 import { PRIMARY_FONT_COLOR } from '../../constants/colors'
+import { SearchIcon } from '@chakra-ui/icons'
 
-export function CourseList({ courses }: { courses: CourseType[] }) {
-  // コースデータ読み込みアニメーション
-  if (!courses) return <Loader />
+export function CourseList({
+  initialCourses,
+}: {
+  initialCourses: CourseType[]
+}) {
+  if (!initialCourses) return <Loader />
+  const [coursesValue, setCoursesValue] = useState(initialCourses)
```

Comment:
```suggestion
  const [courses, setCourses] = useState(initialCourses)
```

じゃだめですかね？
valueとかは極力無くしたいなぁと

Reply:
@issiyda 
value無くしました！